redbot/core/core_commands.py

```diff
@@ -2116,7 +2116,7 @@ async def addadminrole(self, ctx: commands.Context, *, role: discord.Role):
         """
         Adds an admin role for this guild.
 
-        Admins have all the same access and Mods, plus additional admin level commands like:
+        Admins have the same access as Mods, plus additional admin level commands like:
```

Comment:
This one needs to also be updated in `docs/cog_guides/core.rst` file

Reply:
Done